### ISSUE TITLE
Failed to attach to https://website.com/ probably because of enterprise policies.

### ISSUE THREAD

**Issue by alanEG** [google/tamperchrome#199]
hello
Thanks for this job
I am facing a problem when running proxy
This alert appears 
![bandicam 2021-01-15 20-22-49-379](https://user-images.githubusercontent.com/52584554/104763802-23e30900-576f-11eb-8a43-90ff3f7d1384.jpg)

Chrome version:  ` 87.0`
tamperchrome version:  ` 2.0.9`

**Comment by sirdarckcat** [comment]
Hi @alanEG does it happen everywhere? Or just on some websites

**Comment by alanEG** [comment]
hello @sirdarckcat
On all sites

**Comment by sirdarckcat** [comment]
Could you try on a fresh Chrome profile?

It's possible there's another Chrome Extension interfering, and it would be good to find out which one it is.

Thank you!

**Comment by alanEG** [comment]
Thank you @sirdarckcat
The problem was due to some extensions

**Comment by sirdarckcat** [comment]
@alanEG could you comment which extensions? So I can detect it and warn users :-)

**Comment by alanEG** [comment]
These extensions
`Avira Password Manage`
`malwarebytes`

**Comment by alanEG** [comment]
Excuse me
The extension works with `malwarebytes`
The problem was only due to `Avira Password Manage`

**Comment by sirdarckcat** [comment]
Ok thanks! I'll take a look at Avira :-)

**Comment by fbo3264** [comment]
fyi - this also happens if the ModHeaders extension is installed. After removing ModHeaders it works fine.

**Comment by sirdarckcat** [comment]
I think we can fix this, by forcing the user to reload the page, but I need to check.

**Comment by sirdarckcat** [comment]
@fbo3264 I can't reproduce with modheaders. is this this one? https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj/related?hl=en

**Comment by fbo3264** [comment]
@sirdarckcat : Yep, it was that one. The funny thing is I cannot reproduce it any longer either  - very weird.

**Comment by sirdarckcat** [comment]
Quick update.. reloading the page doesn't solve this problem. Any time any extension attaches to a page being debugged, the debugger bails out.

Best solution here is probably warning the user and explaining the culprit.

**Comment by drinks5** [comment]
this also happens if the LastPass extension is installed. After removing LastPass it works fine.

**Comment by Smithfield42** [comment]
Sequence source, script change to support outdated malware
